spec/factories/whitehall_export/index_factory.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :whitehall_export_index, class: Hash do
+    skip_create
+
+    documents do
+      [
+        build(:whitehall_export_index_document),
+      ]
+    end
+
+    page_count { 1 }
```

Comment:
Is the the number of documents? you could use documents.size if so.

Reply:
Sure, updated to use `documents.size`.